iceberg/iceberg-handler/src/test/results/positive/show_partitions_test.q.out

```diff
@@ -0,0 +1,214 @@
+Plan optimized by CBO.
+
+Stage-0
+  Fetch Operator
+    limit:-1
+    Select Operator [SEL_1]
+      Output:["_col0","_col1","_col2","_col3"]
+      TableScan [TS_0]
+        Output:["partition","record_count","file_count","spec_id"]
+
+PREHOOK: query: create table ice2 (a string, b int, c int) PARTITIONED BY (d_part int, e_part int) stored by iceberg stored as orc TBLPROPERTIES("format-version"='2')
+PREHOOK: type: CREATETABLE
+PREHOOK: Output: database:default
+PREHOOK: Output: default@ice2
+POSTHOOK: query: create table ice2 (a string, b int, c int) PARTITIONED BY (d_part int, e_part int) stored by iceberg stored as orc TBLPROPERTIES("format-version"='2')
+POSTHOOK: type: CREATETABLE
+POSTHOOK: Output: database:default
+POSTHOOK: Output: default@ice2
+PREHOOK: query: insert into ice2 values  ('aa', 1, 2, 3, 4), ('aa', 1, 2, 3, 4), ('aa', 1, 2, 2, 5), ('aa', 1, 2, 10, 5), ('aa', 1, 2, 10, 5)
+PREHOOK: type: QUERY
+PREHOOK: Input: _dummy_database@_dummy_table
+PREHOOK: Output: default@ice2
+POSTHOOK: query: insert into ice2 values  ('aa', 1, 2, 3, 4), ('aa', 1, 2, 3, 4), ('aa', 1, 2, 2, 5), ('aa', 1, 2, 10, 5), ('aa', 1, 2, 10, 5)
+POSTHOOK: type: QUERY
+POSTHOOK: Input: _dummy_database@_dummy_table
+POSTHOOK: Output: default@ice2
+PREHOOK: query: select * from default.ice2.partitions order by `partition`
+PREHOOK: type: QUERY
+PREHOOK: Input: default@ice2
+PREHOOK: Output: hdfs://### HDFS PATH ###
+POSTHOOK: query: select * from default.ice2.partitions order by `partition`
+POSTHOOK: type: QUERY
+POSTHOOK: Input: default@ice2
+POSTHOOK: Output: hdfs://### HDFS PATH ###
+{"d_part":10,"e_part":5}	2	1	0
+{"d_part":2,"e_part":5}	1	1	0
+{"d_part":3,"e_part":4}	2	1	0
+PREHOOK: query: show partitions ice2
+PREHOOK: type: SHOWPARTITIONS
+PREHOOK: Input: default@ice2
+POSTHOOK: query: show partitions ice2
+POSTHOOK: type: SHOWPARTITIONS
+POSTHOOK: Input: default@ice2
+{Spec-id=0}*:d_part=10/e_part=5
+{Spec-id=0}*:d_part=2/e_part=5
+{Spec-id=0}*:d_part=3/e_part=4
+PREHOOK: query: ALTER TABLE ice2 SET PARTITION SPEC (c)
+PREHOOK: type: ALTERTABLE_SETPARTSPEC
+PREHOOK: Input: default@ice2
+POSTHOOK: query: ALTER TABLE ice2 SET PARTITION SPEC (c)
+POSTHOOK: type: ALTERTABLE_SETPARTSPEC
+POSTHOOK: Input: default@ice2
+POSTHOOK: Output: default@ice2
+PREHOOK: query: select * from default.ice2.partitions order by `partition`
+PREHOOK: type: QUERY
+PREHOOK: Input: default@ice2
+PREHOOK: Output: hdfs://### HDFS PATH ###
+POSTHOOK: query: select * from default.ice2.partitions order by `partition`
+POSTHOOK: type: QUERY
+POSTHOOK: Input: default@ice2
+POSTHOOK: Output: hdfs://### HDFS PATH ###
+{"d_part":10,"e_part":5,"c":null}	2	1	0
+{"d_part":2,"e_part":5,"c":null}	1	1	0
+{"d_part":3,"e_part":4,"c":null}	2	1	0
+PREHOOK: query: show partitions ice2
+PREHOOK: type: SHOWPARTITIONS
+PREHOOK: Input: default@ice2
+POSTHOOK: query: show partitions ice2
+POSTHOOK: type: SHOWPARTITIONS
+POSTHOOK: Input: default@ice2
+{Spec-id=0}:d_part=10/e_part=5
```

Comment:
why there is no current spec marker?

Reply:
LTER TABLE ice2 SET PARTITION SPEC (c) ;
select * from default.ice2.partitions order by `partition`;
show partitions ice2;
====================
```
POSTHOOK: type: SHOWPARTITIONS
POSTHOOK: Input: default@ice2
{Spec-id=0}:d_part=10/e_part=5
{Spec-id=0}:d_part=2/e_part=5
{Spec-id=0}:d_part=3/e_part=4
```
====================
insert into ice2 values  ('aa', 1, 2, 3, 4), ('aa', 1, 2, 3, 4), ('aa', 1, 3, 2, 5), ('aa', 1, 4, 10, 5), ('aa', 1, 5, 10, 5);
select * from default.ice2.partitions order by `partition`;
show partitions ice2;
====================
```
POSTHOOK: Input: default@ice2
{Spec-id=0}:d_part=10/e_part=5
{Spec-id=0}:d_part=2/e_part=5
{Spec-id=0}:d_part=3/e_part=4
{Spec-id=1}*:c=2
{Spec-id=1}*:c=3
{Spec-id=1}*:c=4
{Spec-id=1}*:c=5
```
===================
The partition shows up in the iceberg.partitions table only after new rows are inserted to the table after the spec has changed.

Updated the PR to the following as discussed:
```
spec-id=0/d_part=10/e_part=5
spec-id=0/d_part=2/e_part=5
spec-id=0/d_part=3/e_part=4
current-spec-id=1/c=2
current-spec-id=1/c=3
current-spec-id=1/c=4
current-spec-id=1/c=5  

```